content/adr/angular-fe.md

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Adopt Angular for Frontend Development'
+type: "adrs"
+date: "05-07-2024"
+decision: "We will adopt Angular as our frontend framework due to its comprehensive tooling, strong typing with TypeScript, and suitability for building large-scale applications."
+status: "proposed"
+categories:
+  - Angular
+---
+
+# ADR for Angular
+
+Below is a list of all Architecture Decision Records (ADRs) for our Angular project.
+
+{{< adr-list >}}
```

Comment:
We shouldn't have this in each ADR file. As such, we don't even need a listing on each ADR. The list should only be there on the main ADR page.